packages/angular-cli/tasks/build.ts

```diff
@@ -32,6 +33,15 @@ export default Task.extend({
           return;
         }
 
+        if (!runTaskOptions.watch && runTaskOptions.json) {
+          const jsonStats = stats.toJson('verbose');
+
+          fs.writeFileSync(
+            path.resolve(project.root, outputDir, 'stats.json'),
```

Comment:
@winkerVSbecks I think `outputDir` should be `outputPath`. There is no variable `outputDir`. That's what the compiler complains about and why the tests fail in ci.